Taskfile.yml

```diff
@@ -47,14 +47,15 @@ tasks:
   build:
     desc: Build main
     cmds:
-      - go build -v -o build/darwin/amd64/circleci .
-  
+      # LDFlags sets the segment endpoint to an empty string so that all it is replaced by the default value
+      - go build -v -o build/darwin/amd64/circleci -ldflags="-X 'github.com/CircleCI-Public/circleci-cli/telemetry.SegmentEndpoint=' -X 'github.com/CircleCI-Public/circleci-cli/telemetry.SegmentKey=AbgkrgN4cbRhAVEwlzMkHbwvrXnxHh35'" .
```

Comment:
I think there is an understandable confusion here: actually, leaving the `SegmentEndpoint` empty lets the `analytics` library fill with the default endpoint. Whereas not giving the value let the `SegmentEndpoint`, leaves the CLI to use the `http://localhost` value instead of the default value.
I admit this is counter-intuitive but I like to have the library fill the default value itself

Reply:
Sounds good. The comment was misleading. I should have looked closer at the code.